player_pyano/app.py

```diff
@@ -9,7 +9,7 @@
 LOCK = False
 SONGS_LIST = os.listdir("songs")
 
-output_name = mido.get_output_names()[0]
+output_name = mido.get_output_names()[1]
```

Comment:
Looks like you can comment on both the old version, and the new version. Here we changed from `[0]` to `[1]`, but probably not for any good reason.